src/reducers/versions.spec.tsx

```diff
@@ -352,6 +352,39 @@ describe(__filename, () => {
       expect(_log.debug).toHaveBeenCalled();
       expect(versionsState).toEqual(previousState);
     });
+
+    it('logs a debug message when headVersion is missing on loadDiff()', () => {
+      const addonId = 1;
+      const baseVersionId = 2;
+      const path = 'some/other/file.js';
+      const _log = createFakeLogger();
+
+      const version = {
+        ...fakeVersionWithDiff,
+        file: {
+          ...fakeVersionWithDiff.file,
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          selected_file: path,
+        },
+      };
+      const headVersionId = version.id;
+
+      const versionsState = reducer(
+        undefined,
+        actions.loadDiff({
+          addonId,
+          baseVersionId,
+          headVersionId,
+          version,
+        }),
+        // TS looks confused about the third optional argument.
```

Comment:
This is odd. Can we do something better with the definition to make TS less confused?

Reply:
No, we can't. This is existing code.

Reply:
This is because TS does not expect a third argument to `reducer` but there is no reason: an optional argument can be added to a function without any problem in JS.